metrics/http.go

```diff
@@ -5,8 +5,17 @@ import (
 	"net/http"
 )
 
-func HTTPHandler(w http.ResponseWriter, r *http.Request) {
-	s := GetStats()
+func GoStatsHandler(w http.ResponseWriter, r *http.Request) {
```

Comment:
__[golint]__
* [golint] exported function GoStatsHandler should have comment or be unexported

<!-- Comment by [SideCI](https://sideci.com) -->